DonjinKrawler-Client/src/main/java/donjinkrawler/PlayerShell.java

```
@@ -12,10 +14,15 @@ public PlayerShell(String name) {
     }
 
     public void loadImage() {
-        ImageIcon ii = new ImageIcon(ClassLoader.getSystemResource("craft.png").getFile());
-        image = ii.getImage();
-        ii = new ImageIcon(ClassLoader.getSystemResource("attack.png").getFile());
-        attackIMG = ii.getImage();
+        try {
+            InputStream stream = getClass().getResourceAsStream("/craft.png");
+            ImageIcon ii = new ImageIcon(ImageIO.read(stream));
+            image = ii.getImage();
+            ii = new ImageIcon(ImageIO.read(getClass().getResourceAsStream("/attack.png")));
+            attackIMG = ii.getImage();
+        } catch (Exception ignored) {
```

If possible fix this.

From the [checklist](https://dzone.com/articles/java-code-review-checklist):
`Dont ignore exceptions.`